pre/remap_restart/remap_params.tpl

```diff
@@ -8,11 +8,16 @@ input:
     drymass: 1
     hydrostatic: 0
   shared:
+    # bc_version info here is not critical. It is used in command_line options
+    bc_version: none
     MERRA-2: false
+    stretch: false
     agrid:
     bcs_dir:
     expid:
     ogrid:
+    # (coupled) ocean model: data, MOM5, MOM6
+    omodel: data
```

Comment:
It probably doesn't matter to python, but the order of the params in this template differs from that in the yaml files for the test cases.  To get a clean start with the remap_restarts.py, it may make sense to re-issue the yaml files for the test cases so they conform with the latest template file before merging the PR into develop.

Reply:
re-ordered